packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -23,6 +23,7 @@
 %global _cachedir %{_localstatedir}/cache
 %global bundled_agent_version %{version}
 %global no_exec_perm 644
+%global debug_package %{nil}
```

Comment:
Q: Is there anything we lost in build logs after adding this?

Reply:
Thanks @yinyic for adding the Debuginfo! 
When we build rpms with internal Koji, the building target platforms is set to `noarch` by default; therefore, debuginfo package is not created according to this [section](https://docs.fedoraproject.org/en-US/packaging-guidelines/Debuginfo/#_missing_debuginfo_packages).

> **Missing debuginfo packages**
It is normal for noarch package builds to not produce a debuginfo package. If it’s missing in other cases (where it has not been explicitly disabled), something’s wrong. One such case is a [missing %build section](https://bugzilla.redhat.com/192422) with some rpmbuild versions.